### PR TITLE
Update to Unity 5.5.1f1

### PR DESCRIPTION
**Pull request by HodgsonSDAS**
Resolves https://github.com/Microsoft/HoloToolkit-Unity/issues/475

https://unity3d.com/unity/whats-new/unity-5.5.1

**Caution!**
> - Warning about missing object references in the inspector after upgraded to 5.5. We have only received one bug report and we have not been able to reproduce the issue. If you find some/all of your object references missing in the inspector do not save your scene. Quit the editor and re-open your project and the references should come back correctly.

HoloLens Specific Improvements:

> - AR: Fixed a crash when exiting play mode during a Holographic Simulation session. (858208)
> - HoloLens: Fixed a crash in Editor when terminating Remoting App during an active Holographic Remoting session.
> - HoloLens: Fixed resume-from-suspend bugs that would occasionally crash the app or not render anything. (800796, 851897, 825951)
> - HoloLens: Improved error reporting when failing to load PerceptionRemotingPlugin.dll (used by Holographic Remoting). (848237)
> - Core: Fixed hangs in the job system when running on certain platforms with a low number of cores e.g. older Windows Phones, UWP dual core machines. (791434)
> - Shaders: Optimized in-editor import, load time and memory usage for shaders with massive amounts of potential variants.

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Shaders/FastConfigurable.shader.meta [hunk 1]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: a82e601d90c076c458969fbd127780e6
+guid: f1f3948b20c230044bda31b620ddd37f
```
Comment (thebanjomatic) [inline Assets/HoloToolkit/Utilities/Shaders/FastConfigurable.shader.meta]: Was this change intentional?

Comment (HodgsonSDAS) [inline Assets/HoloToolkit/Utilities/Shaders/FastConfigurable.shader.meta]: Yes

Comment (thebanjomatic) [inline Assets/HoloToolkit/Utilities/Shaders/FastConfigurable.shader.meta]: so much for globally unique :+1:

Comment (HodgsonSDAS) [inline Assets/HoloToolkit/Utilities/Shaders/FastConfigurable.shader.meta]: Haha the probability is low, but sometimes can happen. Just gotta use really big prime numbers when generating!
